lib/fluent/plugin/out_typecast.rb

```diff
@@ -49,6 +49,8 @@ def emit(tag, es, chain)
 
   def cast_proc(key)
     case key
+    when 'json'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.